README.md

```diff
@@ -21,6 +21,9 @@ go install github.com/alpstable/gidari/cmd/gidari@latest
 
 Gidari is not currently available as a library. Library support will be investigated in [Issue #214](https://github.com/alpstable/gidari/issues/214).
 
+```sh
+go get
```

Comment:
`go get` needs to reference the package, like this:

```sh
go get github.com/alpstable/gidari
```
See [godotenv](https://github.com/joho/godotenv) as a reference. Also can we move this line below the `go get` line: 

> Gidari is not currently available as a stable API. Library support will be investigated in [Issue #214](https://github.com/alpstable/gidari/issues/214).